_std/defines/item.dm

```diff
@@ -49,6 +47,20 @@
 /// If an item cannot be crushed by the crusher
 #define UNCRUSHABLE              (1<<23)
 
+// Defines for is_open_container returns and checking. Don't check them directly, use the procs.
+/// stuff can go in
+#define ISOPEN_INWARD			(1<<24)
+//// stuff can come out
+#define ISOPEN_OUTWARD			(1<<25)
```

Comment:
Yeah no you can't do this, byond only gives us 24 bits to mess with. If you want to do bitflags here you'll have to move it to a separate var.

Reply:
ah i didn't know this

Reply:
i've dropped it to 23 and 24... hopefully that still works

Reply:
moved it to a new var